resources/monitoring/templates/prometheus/rules-1.14/k8s.rules.yaml

```diff
@@ -57,8 +60,8 @@ spec:
           max by(namespace, pod, node) (kube_pod_info{node!=""})
         )
       record: node_namespace_pod_container:container_memory_swap
-    - expr: sum(container_memory_usage_bytes{job="kubelet", metrics_path="/metrics/cadvisor", image!="", container!="POD"}) by (namespace)
-      record: namespace:container_memory_usage_bytes:sum
+    - expr: sum(container_memory_working_set_bytes{job="kubelet", metrics_path="/metrics/cadvisor", image!="", container!="POD"}) by (namespace)
```

Comment:
Are you sure we should change it in the community rules?

Reply:
I would like to make the change for consistency.